airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSourceOperations.java

```diff
@@ -508,7 +520,12 @@ public JsonSchemaType getAirbyteType(final PostgresType jdbcType) {
       case DATE_ARRAY -> JsonSchemaType.builder(JsonSchemaPrimitive.ARRAY)
           .withItems(JsonSchemaType.STRING_DATE)
           .build();
-
+      case JSONB -> JsonSchemaType.builder(JsonSchemaPrimitive.ARRAY,
```

Comment:
Instead of having the builder take multiple values, can you just define a JsonSchemaPrimitive.JSONB (which under the hood will build the same structure/oneof that you've been doing)?

Reply:
> Instead of having the builder take multiple values, can you just define a JsonSchemaPrimitive.JSONB (which under the hood will build the same structure/oneof that you've been doing)?

@akashkulk updated with your proposal